config/prometheus/alert-rules.yml

```diff
@@ -100,7 +100,7 @@ groups:
         annotations:
           error: "Mountpoint {{ $labels.mountpoint }} on {{ $labels.instance }} is over 80% full"
       - alert: DiskWillFillIn2Weeks
-        expr: predict_linear(node_filesystem_free{job="machine-metrics"}[1h], 2w) < 0
+        expr: predict_linear(node_filesystem_free_bytes{job="machine-metrics"}[24h], 2w) < 0
```

Comment:
I wonder if we just delete this alert entirely? The nature of the usage of these machines is probably too lumpy for this to really work - it sounds like it's people downloading a big data set from somewhere, or a model, or generating a mass of data one day, and then it sits idle for a week

Reply:
This alert is applied to all our machines, most of which have pretty steady state usage. I could exclude the GPU machines specifically and keep the rule for all the other machines?

Reply:
right, I see. Let's see how annoying it is, and if it squawks too often we can apply that filter